neon-animated-pages.html

```diff
@@ -94,7 +94,16 @@
 
     _onIronSelect: function(event) {
       var selectedPage = event.detail.item;
-      if (!selectedPage) return;
+
+      // Only consider child elements.
+      if (!(selectedPage instanceof Element)) return;
+      if (Polymer.dom(selectedPage).parentNode != this &&
+          Polymer.dom(selectedPage).getDestinationInsertionPoints().
```

Comment:
An implementor of `Polymer.IronSelectableBehavior` should have an array of its selectable items (`this.items`). Would it be sufficient to do an `indexOf` search of that array instead?